docs/source/user-guide/sql/dialect.md

```diff
@@ -0,0 +1,38 @@
+<!---
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing,
+  software distributed under the License is distributed on an
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either express or implied.  See the License for the
+  specific language governing permissions and limitations
+  under the License.
+-->
+
+# SQL Dialect
+
+By default, DataFusion follows the [PostgreSQL SQL dialect].
+For Array/List functions and semantics, it follows the [DuckDB SQL dialect].
```

Comment:
Is it confusing when an array/list function exists in PostgreSQL but you don't follow it's semantics, even though you do for non array/list functions?

Reply:
I think it is confusing for sure